vms/platformvm/state/state.go

```diff
@@ -2440,3 +2468,27 @@ func (s *state) ReindexBlocks(lock sync.Locker, log logging.Logger) error {
 
 	return s.Commit()
 }
+
+func putFeeState(db database.KeyValueWriter, feeState fee.State) error {
+	feeStateBytes, err := block.GenesisCodec.Marshal(block.CodecVersion, feeState)
+	if err != nil {
+		return err
+	}
+	return db.Put(FeeStateKey, feeStateBytes)
+}
+
+func getFeeState(db database.KeyValueReader) (fee.State, error) {
+	feeStateBytes, err := db.Get(FeeStateKey)
+	if err == database.ErrNotFound {
+		return fee.State{}, nil
```

Comment:
q: shouldn't the capacity be max here? I believe this case happens only before etna is activated. In such a casa shouldn't capacity be max instead of zero?

Reply:
The ACP doesn't specify this... I think that it makes more sense to have it be set to 0 so that the fork activation doesn't have a complexity spike (although I doubt this would really matter).

I'll make a PR to the ACP to clarify this.

Reply:
Added this to the spec: https://github.com/avalanche-foundation/ACPs/pull/136. Feel free to comment on that PR if you think it should be a different value.